addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -35,8 +34,7 @@ ascanbeta.crossdomain.adobe.send.extrainfo = The web server permits malicious cr
 ascanbeta.crossdomain.adobe.send.name = Cross-Domain Misconfiguration - Adobe - Send
 ascanbeta.crossdomain.adobe.send.soln = Configure the crossdomain.xml file to restrict the list of domains that are allowed to make cross-domain send (but not necessarily read) requests to this web server, using <allow-http-request-headers-from domain="example.com" headers="Authorization,X-Blahh">. You should only grant access to "*" (all domains) if you are certain that this service is not vulnerable to Cross Site Request Forgery (CSRF) attacks.
 ascanbeta.crossdomain.name = Cross-Domain Misconfiguration
-#the refs cannot be customised for each sub-category (Adobe, Silverlight, etc.)
-ascanbeta.crossdomain.refs = http://www.adobe.com/devnet/articles/crossdomain_policy_file_spec.html\nhttp://www.adobe.com/devnet-docs/acrobatetk/tools/AppSec/CrossDomain_PolicyFile_Specification.pdf\nhttp://msdn.microsoft.com/en-US/library/cc197955%28v=vs.95%29.aspx\nhttp://msdn.microsoft.com/en-us/library/cc838250%28v=vs.95%29.aspx
```

Comment:
I dropped the comment about sub-cats, they could be. It's 2024 and we should not be encountering flash anymore. That's not 100% sure but it's close. I don't think there's much point splitting the refs up, as this rule should probably be retired &lt;air_quotes&gt;soon&lt;/air_quotes&gt;

Reply:
The file is not just about Flash though.

Reply:
I know it's flash and silver light but I believe SL is rolling or headed roll as well.

Either way the comment is no longer necessary.

Reply:
I mean the Adobe file is not just for Flash, it's for other Adobe programs too (not arguing against just saying).

Agreed.

Reply:
Oh okay, I didn't realize they had other techs using it.